build.sbt

```diff
@@ -30,9 +34,10 @@ lazy val testSettings = List(
   testOptions.in(Test) +=
     Tests.Argument(TestFrameworks.ScalaCheck, "-verbosity", "2"),
   libraryDependencies ++= List(
-    "org.scalatest" %%% "scalatest" % "3.0.5" % Test,
-    "org.scalacheck" %%% "scalacheck" % "1.14.0" % Test,
-    "com.github.alexarchambault" %%% "scalacheck-shapeless_1.13" % "1.1.6" % Test
+    "org.scalatest" %%% "scalatest" % "3.2.0-SNAP10" % Test,
+    //"org.scalacheck" %%% "scalacheck" % "1.14.0" % Test,
+    "com.github.lolgab" %%% "scalacheck" % "1.14.1" % Test, // scala native but no js
```

Comment:
I'm totally fine with using this scalacheck artifact 👍 We could do the same for paiges if necessary, although I think they are trying to get a release out.